### PR TITLE
Update s3 config

### PR DESCRIPTION
**Pull request by LocoRichard**

### CODE REVIEW

**Review by shanghaikid — APPROVED** [review]

**Review by yhmo — APPROVED** [review]

File: site/en/reference/configuration_cluster-advanced.md [hunk 1]
```diff
@@ -76,44 +76,44 @@ MinIO is the storage engine supporting Milvus' data persistence for insert log f
...
+<td>localhost</td>
+</tr>
+<tr>
 		<td><code>minio.port</code></td>
 		<td><details>
-       <summary>Port of MinIO</summary>
+       <summary>Port of MinIO/S3</summary>
        <li>Environment variable: <code>MINIO_ADDRESS</code></li>
-       <li>Access MinIO service with <code>minio.address</code>. <code>minio.address</code> and <code>minio.port</code> together generates the valid access to MinIO.</li>
+       <li>Access MinIO/S3 service with <code>minio.address</code>. <code>minio.address</code> and <code>minio.port</code> together generates the valid access to MinIO/S3.</li>
```
Comment (claireyuw) [inline site/en/reference/configuration_cluster-advanced.md:95]: generate

File: site/en/reference/configuration_cluster-advanced.md [hunk 2]
```diff
@@ -76,44 +76,44 @@ MinIO is the storage engine supporting Milvus' data persistence for insert log f
...
 	</tr>
   <tr>
 		<td><code>minio.AccessKeyID</code></td>
 		<td><details>
-       <summary>MinIO key ID for authorized user access</summary>
+       <summary>MinIO/S3 key ID for authorized user access</summary>
        <li>Environment variable: <code>MINIO_ACCESS_KEY</code></li>
-       <li>Access key ID that MinIO issued to authorized users. <code>minio.accessKeyID</code> and <code>minio.secretAccessKey</code> together is used for identity authentication to access the MinIO service.</li>
-       <li>This configuration must be set identical to the environment variable <code>MINIO_ACCESS_KEY</code>, which is necessary for booting up MinIO. The default value applies to the MinIO service that booted up with the default <b>docker-compose.yml</b> provided by Milvus.</li>
+       <li>Access key ID that MinIO/S3 issued to authorized users. <code>minio.accessKeyID</code> and <code>minio.secretAccessKey</code> together is used for identity authentication to access the MinIO/S3 service.</li>
```
Comment (claireyuw) [inline site/en/reference/configuration_cluster-advanced.md:105]: issues

are used

File: site/en/reference/configuration_standalone-advanced.md [hunk 3]
```diff
@@ -80,44 +80,44 @@ MinIO is the storage engine supporting Milvus' data persistence for insert log f
...
+<td>localhost</td>
+</tr>
+<tr>
 		<td><code>minio.port</code></td>
 		<td><details>
-       <summary>Port of MinIO</summary>
+       <summary>Port of MinIO/S3</summary>
        <li>Environment variable: <code>MINIO_ADDRESS</code></li>
-       <li>Access MinIO service with <code>minio.address</code>. <code>minio.address</code> and <code>minio.port</code> together generates the valid access to MinIO.</li>
+       <li>Access MinIO/S3 service with <code>minio.address</code>. <code>minio.address</code> and <code>minio.port</code> together generates the valid access to MinIO/S3.</li>
```
Comment (claireyuw) [inline site/en/reference/configuration_standalone-advanced.md:99]: generate

File: site/en/reference/configuration_standalone-advanced.md [hunk 4]
```diff
@@ -80,44 +80,44 @@ MinIO is the storage engine supporting Milvus' data persistence for insert log f
...
 	</tr>
   <tr>
 		<td><code>minio.AccessKeyID</code></td>
 		<td><details>
-       <summary>MinIO key ID for authorized user access</summary>
+       <summary>MinIO/S3 key ID for authorized user access</summary>
        <li>Environment variable: <code>MINIO_ACCESS_KEY</code></li>
-       <li>Access key ID that MinIO issued to authorized users. <code>minio.accessKeyID</code> and <code>minio.secretAccessKey</code> together is used for identity authentication to access the MinIO service.</li>
-       <li>This configuration must be set identical to the environment variable <code>MINIO_ACCESS_KEY</code>, which is necessary for booting up MinIO. The default value applies to the MinIO service that booted up with the default <b>docker-compose.yml</b> provided by Milvus.</li>
+       <li>Access key ID that MinIO/S3 issued to authorized users. <code>minio.accessKeyID</code> and <code>minio.secretAccessKey</code> together is used for identity authentication to access the MinIO/S3 service.</li>
```
Comment (claireyuw) [inline site/en/reference/configuration_standalone-advanced.md:109]: issues

are used

File: site/en/reference/configuration_standalone-advanced.md [hunk 5]
```diff
@@ -80,44 +80,44 @@ MinIO is the storage engine supporting Milvus' data persistence for insert log f
...
+       <li>This configuration must be set identical to the environment variable <code>MINIO_ACCESS_KEY</code>, which is necessary for booting up MinIO/S3. The default value applies to the MinIO/S3 service that booted up with the default <b>docker-compose.yml</b> provided by Milvus.</li>
```
Comment (claireyuw) [inline site/en/reference/configuration_standalone-advanced.md:110]: boots up

File: site/en/reference/configuration_standalone-advanced.md [hunk 6]
```diff
@@ -80,44 +80,44 @@ MinIO is the storage engine supporting Milvus' data persistence for insert log f
...
 		<td><details>
-       <summary>MinIO encryption string</summary>
+       <summary>MinIO/S3 encryption string</summary>
        <li>Environment variable: <code>MINIO_SECRET_KEY</code></li>
-       <li>Secret key used to encrypt the signature string and verify the signature string on server. It must be kept strictly confidential and accessible only to the MinIO server and users.</li>
-       <li>This configuration must be set identical to the environment variable <code>MINIO_SECRET_KEY</code>, which is necessary for booting up MinIO. The default value applies to the MinIO service that booted up with the default <b>docker-compose.yml</b> provided by Milvus.</li>
+       <li>Secret key used to encrypt the signature string and verify the signature string on server. It must be kept strictly confidential and accessible only to the MinIO/S3 server and users.</li>
+       <li>This configuration must be set identical to the environment variable <code>MINIO_SECRET_KEY</code>, which is necessary for booting up MinIO/S3. The default value applies to the MinIO/S3 service that booted up with the default <b>docker-compose.yml</b> provided by Milvus.</li>
```
Comment (claireyuw) [inline site/en/reference/configuration_standalone-advanced.md:120]: The default value applies to the MinIO/S3 service that boots up with...

**Review by claireyuw — CHANGES_REQUESTED** [review]

**Pull request merged**